test/createAzureResource.test.ts

```diff
@@ -17,6 +17,9 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
     const testAccount: TestAzureAccount = new TestAzureAccount();
     let oldAdvancedCreationSetting: boolean | undefined;
     const regExpLTS: RegExp = /LTS/g;
+    const resourceName1: string = getRandomHexString().toLowerCase();
+    let webSiteClient: WebSiteManagementClient;
+    const resourceGroupName: string = 'appsvc_rg_windows_centralus';
```

Comment:
You probably don't want to do this.

If you have apps in this resource group and you run this test locally, it could delete all of your web apps.